packages/react/package.json

```diff
@@ -61,7 +61,7 @@
     "lodash.isequal": "^4.5.0",
     "lodash.omit": "^4.5.0",
     "lodash.throttle": "^4.1.1",
-    "react-is": "^16.8.6",
```

Comment:
Will this have any problems with people using React 16? 🤔

Reply:
Out of curiosity, What does `react-is` do?

Reply:
@abbeyhrt It allows us to check if things are a valid React element type. I think the only place we use it is with `isValidElementType()` in the Public API snapshot to assert that components are valid before we add them to the snapshot "queue".

@joshblack I think we're good. There's no changelog that I could find, so I crawled through [the diff between versions](https://github.com/facebook/react/compare/v16.4.0...v17.0.2#) and it looks like there are no breaking changes. I think they bumped the versions of all core packages so they'd be the same values, even if they didn't include breaking changes.

They deprecated two aliases, `ReactIs.isAsyncMode()` and `ReactIs.isConcurrentMode()`. Will be removed in v18. `ReactIs.typeOf()` now supports some new types, `REACT_LAZY_TYPE`, `REACT_MEMO_TYPE`, `REACT_SUSPENSE_TYPE`.